internal/common/tsconfig.bzl

```diff
@@ -161,7 +161,7 @@ def create_tsconfig(ctx, files, srcs,
   # Keep these options in sync with those in playground/playground.ts.
   compiler_options = {
       # De-sugar to this language level
-      "target": "es5" if devmode_manifest or ctx.attr.runtime == "nodejs" else "es6",
+      "target": "es2015",
```

Comment:
I think you need to add back in the ternary so that es6 sources can still be generated. As I understand things es2015 is a strict subset of es6. If I'm wrong about that feel free to tell me :p

Reply:
Yup you were totally right.
Also In putting that back we're also able to maintain backwards compatibility aswell since non of the tests have to be edited